src/lib/eliom_pervasives.client.ml

```diff
@@ -1,6 +1,8 @@
 
 type 'a client_value = 'a Eliom_lib.client_value
 
+type 'a fragment = 'a client_value
```

Comment:
Do we really need a new name? "Fragment" could just as well indicate an `str_item`.